src/main/java/org/folio/entlinks/domain/entity/AuthorityDataStat.java

```diff
@@ -39,7 +41,7 @@ public class AuthorityDataStat extends AuditableEntity implements Identifiable<U
 
   @ToString.Exclude
   @ManyToOne
-  @JoinColumn(name = "authority_id")
+  @JoinColumn(name = "authority_id", foreignKey = @ForeignKey(name = "none", value = ConstraintMode.NO_CONSTRAINT))
```

Comment:
What is this for?

Reply:
The actual ```authority_id``` FK was dropped from data stat table in this [PR](https://github.com/folio-org/mod-entities-links/pull/187). However, in integration tests I was seeing that hibernate makes joins in queries and it is kinda a expected as hibernate does not know if the physical FK exists or not. So, in order not to have join queries I wanted to solve this question in that way, but I guess eventually this solution also did not help, I will double check it again.